src/main/resources/ctp/types/payment-paypal.json

```diff
@@ -0,0 +1,215 @@
+{
+  "key": "payment-paypal",
```

Comment:
Do you have this field `paymentMethodInfo`? https://github.com/commercetools/commercetools-paypal-plus-integration/pull/109/files#diff-2cd7e972a9ecd144aeb5128d052e7e97R56

Reply:
it is not a field, it is native payment property

Reply:
But in the link, you wrote: `Required fields for the CTP payment object` 😕

Reply:
yes, this payment field must be set in the payment before calling `handle`

Reply:
But I don't see the field definition for `paymentMethodInfo` in this paypal custom type file. Is it ok?

Reply:
yes, because `paymentMethodInfo` **is not a custom type**, it is a native Payment property:

http://dev.commercetools.com/http-api-projects-payments.html#paymentmethodinfo